conf/deployment/aws/ipi_6az_rhcos_3m_6w.yaml

```diff
@@ -0,0 +1,22 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: true
+  ocs_operator_nodes_to_label: 6
+ENV_DATA:
+  platform: 'aws'
+  deployment_type: 'ipi'
+  region: 'us-east-1'
+  worker_availability_zones:
+    - 'us-east-1a'
+    - 'us-east-1b'
+    - 'us-east-1c'
+    - 'us-east-1d'
+    - 'us-east-1e'
+    - 'us-east-1f'
+  master_availability_zones:
+    - 'us-east-1a'
+    - 'us-east-1b'
+    - 'us-east-1c'
+  worker_replicas: 6
+  master_replicas: 3
+  worker_instance_type: 'm4.2xlarge'
```

Comment:
Add a comment that m5.4xlarge is not supported in - 'us-east-1e' zone please.

Reply:
Done